apps/www/src/app/logo.svg

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this file not used?

Reply:
Both  star.svg and logo.svg are not referenced anywhere, but in case of doubt we could leave them here, in the same way as Roni's picture.